nixos/modules/services/networking/zerotierone.nix

```diff
@@ -17,6 +17,15 @@ in
     '';
   };
 
+  options.services.zerotierone.port = mkOption {
```

Comment:
I think a refactor of these 4 options under `options.services.zerotierone` would be nice, other than that this looks good to me.